### PR TITLE
Use mcrouter/gencache

### PR DESCRIPTION
**Pull request by bsimpson63**
* Change the cache key to use a mcrouter routable prefix
* Use stalecache

👓 @dwick

### CODE REVIEW

File: reddit_adzerk/lib/cache.py [hunk 1]
```diff
@@ -4,22 +4,21 @@
...
 
     @classmethod
     def get(cls, flight_id):
-        fullname = g.cache.get(cls._cachekey(flight_id))
+        fullname = g.gencache.get(cls._cache_key(flight_id), stale=True)
 
         if not fullname:
             q = PromoCampaign._query(
                 PromoCampaign.c.external_flight_id == flight_id,
-                data=True,
```
Comment (dwick) [inline reddit_adzerk/lib/cache.py]: isn't `data=True` required when querying on attributes not in `Thing`?

**Comment by dwick** [comment]
💅 one question

File: reddit_adzerk/lib/cache.py (hunk 1 above)
Comment (bsimpson63) [inline reddit_adzerk/lib/cache.py]: Not anymore. We now always load the data attributes and `data=True` is meaningless.

**Comment by bsimpson63** [comment]
💇

**Comment by dwick** [comment]
🐟

File: reddit_adzerk/lib/cache.py (hunk 1 above)
Comment (dwick) [inline reddit_adzerk/lib/cache.py]: 🔪 🔪 🔪 🔪 🔪 🔪

Comment (zeantsoi) [inline reddit_adzerk/lib/cache.py]: > We now always load the data attributes and `data=True` is meaningless.

@bsimpson63, is this true of everything that subclasses `Thing`?

Comment (bsimpson63) [inline reddit_adzerk/lib/cache.py]: Yeah, `data=True` is meaningless everywhere and can be removed.

**Comment by bsimpson63** [comment]
merged

**Pull request closed without merging**